### PR TITLE
Mock pyth product

### PR DESCRIPTION
**Pull request by partiallysorted**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -7,7 +7,7 @@
     "lint": "eslint . --ext ts",
     "prepare": "husky install",
     "precommit": "npm run compile && npm run fmt:fix && npm run lint && npm test",
-    "test": "anchor test --skip-lint"
+    "test": "anchor test --skip-lint -- --features devnet"
```
Comment (jetjake) [inline package.json:10]: Same question as the other PR, just wondering what that argument means.

File: programs/mock-pyth/src/lib.rs [hunk 2]
```diff
@@ -0,0 +1,289 @@
...
+        priceData.agg.status = PriceStatus::Trading;
+
+        priceData.twap = price;
+        priceData.expo = expo;
+        priceData.ptype = PriceType::Price;
+
+        Ok(())
+    }
+
+    pub fn update_price(ctx: Context<UpdatePrice>, price: i64, conf: u64) -> Result<()> {
```
[the remark this answers is not in the document]
Comment (qiqicodes) [reply programs/mock-pyth/src/lib.rs:69]: I think this will help my current task as well.

[the remark this answers is not in the document]
Comment (partiallysorted) [reply programs/mock-pyth/src/lib.rs:69]: The Pyth product account is created in initialize.

File: package.json (hunk 1 above)
Comment (partiallysorted) [reply package.json:10]: It can also be used to give the mainnet program a different program id than devent.

**Pull request merged**